packages/shared/prisma/schema.prisma

```diff
@@ -2,948 +2,962 @@
+  id                  String                 @id @default(cuid())
+  orgId               String                 @map("org_id")
+  createdAt           DateTime               @default(now()) @map("created_at")
+  updatedAt           DateTime               @default(now()) @updatedAt @map("updated_at")
+  name                String
+  projectMembers      ProjectMembership[]
+  organization        Organization           @relation(fields: [orgId], references: [id], onUpdate: Cascade, onDelete: Cascade)
+  traces              Trace[]
+  observations        Observation[]
+  apiKeys             ApiKey[]
+  dataset             Dataset[]
+  RawEvents           Events[]
+  invitations         MembershipInvitation[]
+  sessions            TraceSession[]
+  Prompt              Prompt[]
+  Model               Model[]
+  EvalTemplate        EvalTemplate[]
+  JobConfiguration    JobConfiguration[]
+  JobExecution        JobExecution[]
+  LlmApiKeys          LlmApiKeys[]
+  PosthogIntegration  PosthogIntegration[]
+  Score               Score[]
+  scoreConfig         ScoreConfig[]
+  BatchExport         BatchExport[]
+  comment             Comment[]
+  annotationQueue     AnnotationQueue[]
+  annotationQueueItem AnnotationQueueItem[]
+
+  @@index([orgId])
+  @@map("projects")
 }
 
 model ApiKey {
-    id                  String    @id @unique @default(cuid())
-    createdAt           DateTime  @default(now()) @map("created_at")
-    note                String?
-    publicKey           String    @unique @map("public_key")
-    hashedSecretKey     String    @unique @map("hashed_secret_key")
-    fastHashedSecretKey String?   @unique @map("fast_hashed_secret_key")
-    displaySecretKey    String    @map("display_secret_key")
-    lastUsedAt          DateTime? @map("last_used_at")
-    expiresAt           DateTime? @map("expires_at")
-    projectId           String    @map("project_id")
-    project             Project   @relation(fields: [projectId], references: [id], onDelete: Cascade)
-
-    @@index(projectId)
-    @@index(publicKey)
-    @@index(hashedSecretKey)
-    @@index(fastHashedSecretKey)
-    @@map("api_keys")
+  id                  String    @id @unique @default(cuid())
+  createdAt           DateTime  @default(now()) @map("created_at")
+  note                String?
+  publicKey           String    @unique @map("public_key")
+  hashedSecretKey     String    @unique @map("hashed_secret_key")
+  fastHashedSecretKey String?   @unique @map("fast_hashed_secret_key")
+  displaySecretKey    String    @map("display_secret_key")
+  lastUsedAt          DateTime? @map("last_used_at")
+  expiresAt           DateTime? @map("expires_at")
+  projectId           String    @map("project_id")
+  project             Project   @relation(fields: [projectId], references: [id], onDelete: Cascade)
+
+  @@index(projectId)
+  @@index(publicKey)
+  @@index(hashedSecretKey)
+  @@index(fastHashedSecretKey)
+  @@map("api_keys")
+}
+
+model BackgroundMigration {
```

Comment:
maybe we should add a short comment here for the purpose and that this table is not related to any project / org

Reply:
@maxdeichmann I wonder why we would need to make that explicit? Theoretically, no table at all is org/project specific. They may have a column that indicates org/project membership for a row, but tables itself are always instance-entities.

Reply:
This was just a thought. Usually all the tables have project_id, org_id. But the readme is very extensive as well. Good enough to keep it the way it is.